tests/libvmi/status.go

```diff
@@ -38,3 +38,15 @@ func GetPodByVirtualMachineInstance(vmi *v1.VirtualMachineInstance, namespace st
 
 	return controlledPod
 }
+
+func GetInterfaceStatusByName(vmi *v1.VirtualMachineInstance) (map[string]v1.VirtualMachineInstanceNetworkInterface, error) {
```

Comment:
This hints it is a getter, but it is an indexer.
Can you replace `Get` with `Index` ?

Reply:
Done